src/gui/properties/propertieswidget.cpp

```diff
@@ -81,6 +81,9 @@ PropertiesWidget::PropertiesWidget(QWidget *parent)
 
     m_state = VISIBLE;
 
+    // Properties list
```

Comment:
Why is it called **Properties list**?

Reply:
Don't know, i used the name currently used.
`TorrentContentFilterModel *m_propListModel;`

Maybe we should cleanup the distinction from properties widget and content tab. In a different PR.

Reply:
>Maybe we should cleanup the distinction from properties widget and content tab. In a different PR.

Ok. But don't need to produce this code comment here.

Reply:
Changed to `Files list`